aten/src/ATen/Version.cpp

```diff
@@ -186,6 +186,9 @@ std::string show_config() {
 }
 
 std::string get_cxx_flags() {
+  #if defined(FBCODE_CAFFE2)
+  TORCH_CHECK(false, "CXX_FLAGS is only for OSS build.");
```

Comment:
This error message is not as descriptive as it could be. Why is CXX_FLAGS only for OSS build?

Reply:
```Buck does not populate the `CXX_FLAGS` field of Caffe2 build options. As a result, get_cxx_flags is OSS only.```?